eZ/Publish/Core/settings/search_engines/legacy/criterion_handlers_common.yml

```diff
@@ -33,10 +33,10 @@ parameters:
         stopWordThresholdFactor: 0.66
         enableWildcards: true
         commands:
+            - "ascii_search_cleanup"
             - "apostrophe_normalize"
             - "apostrophe_to_doublequote"
             - "ascii_lowercase"
-            - "ascii_search_cleanup"
```

Comment:
`ascii_search_cleanup`: if I got the summary right this morning we want to keep the change here, and you should then get rid of `apostrophe_normalize` and `apostrophe_to_doublequote` as you say.

> At first I thought TransformationProcessor might be common

`TransformationProcessor` itself is used several places, but the config you change is only injected into WordIndexer. So is there something we need to cleanup there? Are the config changes done here also relevant for other uses of `TransformationProcessor`_(mainly UrlAlias and FullText criterion)_